assets/css/need-a-sweet-heart.css

```diff
@@ -0,0 +1,12 @@
+.need-a-sweet-heart-wrapper{
+    height:80vh;
+    width:100%;
+    background:#E0e7ef;
+}
+.need-a-sweet-heart-photo-heading{
+    background-image: url('../images/the-dog-white-and-black.jpg');
```

Comment:
Element nazywa się heading i ma background-image? Coś tu nie gra

Reply:
Do wyrzucenia. Heading to nie miejsce na background image, a na pewno nie w tym przypadku